library/src/main/java/com/schibsted/spain/barista/BaristaClickActions.java

```diff
@@ -25,64 +28,82 @@
 
 public class BaristaClickActions {
 
-  private static final ResourceTypeChecker RESOURCE_TYPE_CHECKER = new ResourceTypeChecker();
+    private static final ResourceTypeChecker RESOURCE_TYPE_CHECKER = new ResourceTypeChecker();
```

Comment:
I think you accidentally changed the indentation of this file. In this project we use a 2-spaces indentation. Please, could you revert it?